rr_evgp/conf/planner_sim_hillclimb.yaml

```diff
@@ -0,0 +1,30 @@
+collision_hitbox:
+    front: 1.5
+    back: 0.1
+    side: 0.65
+
+max_speed: 5.0
+max_drive_accel: 10
+
+wheel_base: 0.97
+lateral_accel: 2.0
+max_steering: 0.3
+steering_speed: 1000
+steering_gain: 1.0
```

Comment:
Should this be added to every planner for IARRC as well? 
Because `roslaunch rr_iarrc test_circuit_sim.launch` with `roslaunch rr_gazebo macaroni_iarrc_circuit.launch` make the car drive off the course on the first turn now.

Reply:
Lol, the circuit planner on master also seems to be broken...

Reply:
Nvm, all is fixed when I update the laplacian line sim detector